docs/SRS-Volere/SRS.tex

```diff
@@ -722,9 +722,21 @@ \section{Costs}
 
 \section{User Documentation and Training}
 \subsection{User Documentation Requirements}
-\lips
+\begin{enumerate} [{UDT-DR}1.]
+\item A user manual is required which outlines a procedure for the stakeholder
+use cases. This is needed for the facilities managers user group.
+\item A set-up manual specifying how both city employees and contractors
+can create an account in the system. This manual should include
+troubleshooting techniques for common problems. 
+\end{enumerate}
+
 \subsection{Training Requirements}
-\lips
+\begin{enumerate} [{UDT-TR}1.]
+\item A training session will need to be hosted to demonstrate the
+functionality of the final product. In particular, it must show how each of 
+the facilities managers use cases are performed. The training must available to
+anyone the City feels would benefit from this trianing.
```

Comment:
what is the format of this training? you mention a training session, but I figure we would just make a pre-recorded video,  because we're not going to be around for training sessions

Reply:
Sure, if we prefer a video I can change it to that. I imagine whenever our last meeting with the City is next year we'll be showing them what we demo at the capstone expo, so that might be considered a training session if we show all the features and how to use them

Reply:
changed to video